spring-integration-smb/src/test/java/org/springframework/integration/smb/outbound/SmbSendingMessageHandlerTests.java

```diff
@@ -62,7 +62,7 @@ public void prepare() {
 		smbSessionFactory.setUsername("sambaguest");
 		smbSessionFactory.setPassword("sambaguest");
 		smbSessionFactory.setShareAndDir("smb-share/");
-		//smbSessionFactory.setReplaceFile(true);  // deprecated as of 1.3
+		//smbSessionFactory.setReplaceFile(true);  // deprecated as of 1.2.2
```

Comment:
Why do you still keep this code over here?
I really see it only removed altogether.
Having it commented out does not give us too much value.

Reply:
removed.